client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/test/index.tsx

```diff
@@ -82,7 +82,11 @@ describe( 'SiteMigrationInstructions', () => {
 	it( 'should render preview column', async () => {
 		const { container } = render();
 
-		expect( container.querySelector( '.launchpad-container__main-content' ) ).toBeInTheDocument();
+		expect(
```

Comment:
We should use an aria label to describe the preview panel, something like:
<div aria-label= {translate("Preview image from my")}>

and 
`getByLabelText`